draft-eggert-bcp45bis.md

```diff
@@ -203,6 +212,8 @@ This document does not raise any security issues.
 
 This document does not request any IANA actions.
 
+**RW: Should RFC3005 be an informative reference, given that this document obsoletes it?**
+
```

Comment:
```suggestion
**RW: Should RFC3005 be an informative reference, given that this document obsoletes it?**
```
It's already a normative reference?

Reply:
As in, I was suggesting that it should be an informative reference rather than a normative reference.  Given that this document obsoletes RFC 3005, it shouldn't be necessary to understand RFC 3005 to be able to understand this document.

Reply:
Ah. Can do!